core/src/main/java/org/kiwix/kiwixmobile/core/settings/SliderPreference.java

```diff
@@ -40,6 +43,9 @@
 
   private TextView mMessage;
 
+  @Inject
```

Comment:
If you annotate a field `@Inject` then you have to call `TheCorrectComponent.inject(this)` for the object to be injected.

In this instance it would be `CoreApp.getCoreComponent().inject(this)` and that should be called in the constructor